<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>537</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>start</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/TicTacToe_icon.png</normaloff>icon/TicTacToe_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 43, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>100</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;YEKAN PLUS&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>welcome to TicTacToe game</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="player_CPU">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>player vs CPU</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="player_player">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>player vs player</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
